muni_portal/settings.py

```diff
@@ -205,3 +210,51 @@
 # "S3Boto3Storage does not correctly handle duplicate filenames in their default configuration."
 # https://docs.wagtail.io/en/v2.7.1/advanced_topics/deploying.html
 AWS_S3_FILE_OVERWRITE = False
+
+
+DEFAULT_FROM_EMAIL = env.str("DEFAULT_FROM_EMAIL", None)
+
+
+# https://www.django-rest-framework.org/api-guide/settings/
+REST_FRAMEWORK = {
+    "DEFAULT_AUTHENTICATION_CLASSES": (
+        "rest_framework_simplejwt.authentication.JWTAuthentication",
+    )
+}
+
+
+# https://django-rest-registration.readthedocs.io/en/latest/quickstart.html#preferred-configuration
+REST_REGISTRATION = {
+    "AUTH_TOKEN_MANAGER_CLASS": "muni_portal.core.auth.RestFrameworkAuthJWTTokenManager",
```

Comment:
why are we customising this?

Reply:
Because `rest_registration` doesn't support JWT tokens.
Check limitation section here https://pypi.org/project/django-rest-registration/